twigs/fingerprint.py

```diff
@@ -69,5 +69,22 @@ def get_inventory(args):
             if len(ssh_issues) != 0:
                 asset_data['tags'].append('SSH Audit')
             asset_data['config_issues'] = ssh_issues
+        if args.wordpress == True:
+            word = [NMAP + ' -sV --script http-wordpress-enum '+ args.hosts]
+            try:
+                word_out = subprocess.check_output(word, shell=True)
+            except subprocess.CalledProcessError:
+                logging.error("Error determining OS release")
+                return None 
+            plugins = str(word_out).split('plugins')[-1]
+            plugins = plugins.split('_http')[0]
+            plugins = plugins.split('\\n')
+            for p in range(len(plugins)):
+                plugins[p] = plugins[p].replace(' ','')
+                plugins[p] = plugins[p].replace('|','')
+            plugins = [i for i in plugins if i]
+            asset_data['wordpress_plugins'] = plugins
```

Comment:
plugin information should be added as products like this

        asset_data = {}
        asset_data['id'] = addr
        asset_data['name'] = hostname
        asset_data['type'] = 'WordPress'
        asset_data['owner'] = args.handle
        asset_data['products'] = products
        asset_data['tags'] = asset_tags

Here 'products' is the list of wordpress_plugins and tags is also a list which can contain a single tag 'wordpress' for now.
'type' should be set to 'WordPress' which will help identify and run assessment for this asset.